frontend/src/components/FilterPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import FilterButton from './FilterButton'
+import FilterCategory from './FilterCategory'
+
+const FILTERS = [
+  {
+    category: 'Dietary Restrictions',
+    options: ['Vegan', 'Vegetarian', 'Gluten Free', 'Kosher'],
+  },
+  {
+    category: 'Cuisines',
+    options: [
+      'American',
+      'Italian',
+      'Fast Food',
+      'Pizza',
+      'Mexican',
+      'Asian',
+      'Tacos',
+      'Mediterranean',
+      'Sushi',
+    ],
+  },
+  { category: 'Budget', options: ['$', '$$', '$$$', '$$$$'] },
+  { category: 'Ratings', options: ['2+', '3+', '4+', '5+'] },
+]
+
+export default function FilterPanel() {
+  return (
+    <div className="w-1/5 px-2 bg-white text-black">
+      <h1 className="mb-4 text-4xl font-bold">Your Preferences</h1>
+      {FILTERS.map((filter, idx) => (
+        <FilterCategory
+          key={idx}
+          title={filter.category}
+          options={filter.options}
+        />
+      ))}
+      <button
+        type="submit"
```

Comment:
Nit: We can omit the `type="submit"` since this button isn't attached to a `form` element